java/java-impl/src/com/intellij/refactoring/introduceParameterObject/ParameterObjectBuilder.java

```diff
@@ -70,9 +70,9 @@ public void setFile(@NotNull PsiFile file) {
 
   public String buildBeanClass() {
         @NonNls final StringBuffer out = new StringBuffer(1024);
-        if (packageName.length() > 0) out.append("package " + packageName + ';');
+        if (packageName.length() > 0) out.append("package ").append(packageName).append(';');
```

Comment:
IntelliJ threw warnings about having concatenated strings within an `append`, and preferred sequential appends.